doc/sphinx/run.rst

```diff
@@ -46,13 +45,13 @@ would like to recompute the forces, despite the fact that they are
 already correctly calculated. To this aim, the option can be used to
 enforce force recalculation.
 
-``minimize_energy``: Run steepest descent minimization
```

Comment:
Why not to have the command in the heading?

Reply:
To my understanding the command names are part of the inline documentation.